test/Interop/Cxx/namespace/Inputs/module.modulemap

```diff
@@ -0,0 +1,7 @@
+module Namespaces {
+  header "namespaces.h"
+}
+
+module NamespaceDefs {
```

Comment:
What does 'defs' mean and why does the first module not have it in the name? I'd suggest to name the two modules `Namespaces1` and `Namespaces2`, they seem to be siblings in the sense that one is not more important than the other.

Reply:
Yeah, I fixed this. I meant to include the one header in the other so that we could test with two headers (because then the decls sometimes become unlinked, i.e., redecls isn't all the decls). Do the names make sense now?